src/components/BookSearchPage/BookCard.tsx

```diff
@@ -0,0 +1,93 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { Box, Typography, Rating, Stack } from '@mui/material';
+
+interface BookCardProps {
+  title: string;
+  author: string;
+  cover: string;
+  customerReviewRank: number;
+  priceStandard: number;
+  link: string; // 필수 link 속성 추가
+}
+
+const BookCard = ({
+  title,
+  author,
+  cover,
+  customerReviewRank,
+  priceStandard,
+  link,
+}: BookCardProps): JSX.Element => {
+  return (
+    <Box
+      component="a"
+      href={link}
+      target="_blank"
+      rel="noopener noreferrer"
+      css={css`
+        display: flex;
+        flex-direction: column;
+        width: 150px;
+        height: 100%;
+        padding: 16px;
+        border: 1px solid #e0e0e0;
+        border-radius: 8px;
+        cursor: pointer;
+        text-decoration: none;
+        color: inherit;
+        transition: box-shadow 0.3s ease;
+        &:hover {
+          box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
+        }
+      `}
+    >
+      <img
+        src={cover}
+        alt={title}
+        css={css`
+          width: 100%;
+          height: 50%;
+          object-fit: cover;
+          border-radius: 4px;
+        `}
+      />
+      <Stack direction="column" spacing={1} sx={{ mt: 1 }}>
```

Comment:
이건 선호도 측면인 것 같아서 반영 안 하셔도 괜찮습니다!

```suggestion
      <Stack direction="column" spacing={1} marginTop={1}>
```

저는 props로 넘길 수 있는 부분은 sx로 스타일 덮어쓰기 보다는 prop으로 넘겨줘서 이 스타일을 사용하겠다고 전달하는 걸 선호하기는 합니다 😊